Explicit version output

msalou opened the pull request.

kauzlari commented:
Please add the possibility of using "./sympler --version" in the output of "./sympler" and "./sympler --help"

msalou commented:
Added possibility of using "./sympler --version" in the output of "./sympler" and "./sympler --help". Further, added time of last commit to version number.

kauzlari commented:
BUG: 
I think I broke your code by removing the "-" from the directory name "sympler-Freesteam" extracted from a zip. Please check that the CMAKE script does s.th. meaningful in this case.

INPUT:

$ CXXFLAGS="-O3" cmake /stuff/codes/codeSympler_msalouFreesteamFromZip/symplerFreesteamJustAnotherCopyWithCorruptedDirName/

OUTPUT:

...
-- Execute version control tracking
CMake Error at CMakeLists.txt:196 (MATH):
  math cannot parse the expression: "-1+1": syntax error, unexpected
  exp_MINUS, expecting exp_OPENPARENT or exp_NUMBER (1)


CMake Error at CMakeLists.txt:198 (string):
  string begin index: -1 is out of range 0 - 51


-- Configuring incomplete, errors occurred!
